addons/binding/org.openhab.binding.kodi/ESH-INF/i18n/kodi_de.properties

```diff
@@ -1,50 +1,55 @@
 # binding
 binding.kodi.name = Kodi Binding
-binding.kodi.description = Das Kodi Binding erlaubt das Kodi Media Center zu steuern 
+binding.kodi.description = Dieses Binding integriert das Kodi Media Center. Durch diese kann das Media Center gesteuert werden.
 
 # thing types
 thing-type.kodi.kodi.label = Kodi Media Center
-thing-type.kodi.kodi.description = Kodi Media Center Binding
-thing-type.config.kodi.kodi.ipAddress.label = Netzwerkaddresse
-thing-type.config.kodi.kodi.ipAddress.description = IP Addresse oder Host Name des Kodi Media Centers
+thing-type.kodi.kodi.description = Kodi Media Center.
+
+# thing types config
+thing-typ.property.kodi.kodi.version = Version
+
+thing-type.config.kodi.kodi.ipAddress.label = IP Adresse
+thing-type.config.kodi.kodi.ipAddress.description = Lokale IP Adresse oder Hostname des Kodi Media Centers.
 thing-type.config.kodi.kodi.port.label = Port
-thing-type.config.kodi.kodi.port.description = Port des Web Socket Services des Kodi Media Centers
+thing-type.config.kodi.kodi.port.description = Port des Kodi Media Centers.
 
 # channel types
 channel-type.kodi.volume.label = Lautst�rke
-channel-type.kodi.volume.description = Lautst�rke des Kodi Media Centers
+channel-type.kodi.volume.description = Erm�glicht die Steuerung der Lautst�rke.
 channel-type.kodi.mute.label = Stumm schalten
-channel-type.kodi.mute.description = Das Kodi Media Centers stumm schalten
-channel-type.kodi.control.label = Kontrolle
-channel-type.kodi.control.description = Steuert das Kodi Media Center (z.B. Play, Pause, Stop)
+channel-type.kodi.mute.description = Erm�glicht die Lautst�rke auf stumm zu schalten.
+channel-type.kodi.control.label = Fernbedienung
+channel-type.kodi.control.description = Erm�glicht die Steuerung der Wiedergabe von Medien auf dem Kodi Media Center (z.B. Play, Pause).
 channel-type.kodi.stop.label = Stop
-channel-type.kodi.stop.description = Stopt die Wiedergabe
-channel-type.kodi.playuri.label = URI abspielen
-channel-type.kodi.playuri.description = Spielt einen URI ab
-channel-type.kodi.pvr-open-tv.label = PVR TV Kanal abspielen
-channel-type.kodi.pvr-open-tv.description = Spielt einen PVR TV Kanal ab
-channel-type.kodi.pvr-open-radio.label = PVR Radio Kanal abspielen
-channel-type.kodi.pvr-open-radio.description = Spielt einen PVR Radio Kanal ab
+channel-type.kodi.stop.description = Erm�glich die Wiedergabe zu stoppen.
```

Comment:
"Ermöglicht das Stoppen der Wiedergabe"
Same syntax you used in the following descriptions.